### ISSUE TITLE
Model Builders Advanced Is this tip relevant?

### ISSUE THREAD

**Issue by domsinclair** [xenial-io/Xenial.Framework#33]
Immediately above the heading Type Safe is the following Tip

TIP

We use that internally for example to combine the visibility properties like NotVisibleInAnyView().
See Built-in Attributes for more information.

#Type safe

I can't help but think that this is a case where something came into your mind at that point and you jotted it down, but I can't see how it is relevant here,  It could easily be my own ignorance but I've been struggling with this ever since I first read it.

**Comment by biohazard999** [comment]
I agree. I think it does not help or clarify anything here. 

My intention was: there arn't a lot of use cases to combine properties for general purpuse attributes, but `NotVisibleInAnyView()` is one of them, so i thought it illustrates another use case, besides domain types.

Maybe we can work out a second sample that can show the power of combining attributes like `[Size(200)], [RuleRequiredField] and [RuleStringLenght(200)]` for example?

**Comment by domsinclair** [comment]
Agreed, expanding on the potential of of what can be done makes perfect sense.
